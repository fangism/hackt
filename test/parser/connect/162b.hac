// "162b.hac"
// accepts-invalid test case

defproc bar(bool p, q) { }

defproc foo(bool a, b) {
bar AP(2, 4);	// invalid, cannot pass parameter to port, duh!
}


