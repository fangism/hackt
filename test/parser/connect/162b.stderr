Error in port actuals, argument 1 is not a physical instance.  [7:8]
ERROR in object_list produced at [7:7..12]
